### PR TITLE
Add media section to about

### PR DESCRIPTION
**Pull request by adamdmharvey**
Add links to some recent media presentations about the doco to the about page.

To reduce duplication, introduced Chris' subtitle as per the GitHub Universe intro, and then avoided it for subsequent bullets. But happy to take refactoring/formatting.

### CODE REVIEW

**Comment by adamdmharvey** [comment]
cc @cpswan @tom-halpin @ojacques

**Comment by cpswan** [comment]
LGTM

File: about/index.md [hunk 1]
```diff
@@ -26,6 +26,21 @@ they will be useful. Our primary goal is not to offer trainings as a service,
...
+  * Presentation by [Chris Swan](https://github.com/cpswan), CTO Modern Apps & Cloud Native, DXC Fellow, DXC Technology
+
+* [All Day DevOps 2020: Open Source Online DevOps Dojo](https://content.sonatype.com/2020addo/addo2020-ct-swan-halpin) (2020 Nov 12)
+  * Presentation by [Chris Swan](https://github.com/cpswan) & [Tom Halpin](https://github.com/tom-halpin), DXC DevOps Technical Lead
+
+* [DXC TechTalk: Online DevOps Dojo, Part 1](https://www.dxc.technology/innovation/events/148828-dxc_techtalk_online_devops_dojo) (2020 May 19)
+  * Presentation by [Chris Swan](https://github.com/cpswan) & [Olivier Jacques](https://github.com/ojacques), _DXC Distinguished Technologist_
+
+* [GitHub Customer Stories: DXC Technology & GitHub](https://github.com/customer-stories/dxc)
+
```
Comment (ojacques) [inline about/index.md:43]:
```suggestion
* [GitHub Customer Stories: DXC Technology & GitHub](https://github.com/customer-stories/dxc)

* [InfoQ interview: DevOps Dojo Provides Online, Interactive DevOps Training](https://www.infoq.com/news/2020/06/devops-dojo/)
```

**Review by ojacques — APPROVED** [review]
Looks good to me.

**Pull request merged**